acceptance/cookbook-git/.kitchen.ec2.yml

```diff
@@ -0,0 +1,300 @@
+# Not quite ready yet
+
+<%
+def file_if_exists(path)
+  path = File.expand_path(path)
+  File.exist?(path) ? path : nil
+end
+%>
+
+driver:
+  name: ec2
+  tags:
+    X-Project: Kitchen Tests
+  aws_ssh_key_id: <%= ENV['AWS_SSH_KEY_ID'] || ENV['USER'] %>
+  transport:
+    ssh_key: <%= file_if_exists("~/.ssh/#{ENV['AWS_SSH_KEY_ID'] || ENV['USERNAME']}.pem") ||
+                 file_if_exists("~/.ssh/#{ENV['AWS_SSH_KEY_ID'] || ENV['USERNAME']}")
+                 file_if_exists("~/.ssh/id_rsa") %>
+  # test-specific stuff
+  region: us-west-2
+  availability_zone: a
+  subnet_id: subnet-19ac017c
+  security_group_ids: ["sg-e401eb83", "sg-96274af3"]
+  instance_type: m3.large
+  associate_public_ip: true
+
+platforms:
+  #
+  # AIX
+  #
+  # - name: aix-6.1
+  # - name: aix-7.1
+  #
+  # Debian
+  #
+  - name: debian-8
+    driver:
+      image_search:
+        name: debian-jessie-*
+        owner-id: 379101102735
+        architecture: x86_64
+        virtualization-type: hvm
+        block-device-mapping.volume-type: gp2
+        image-type: machine
+      instance_type: t2.micro
+    transport:
+      username: admin
+  - name: debian-7
+    driver:
+      image_search:
+        name: debian-wheezy-*
+        owner-id: 379101102735
+        architecture: x86_64
+        virtualization-type: hvm
+        block-device-mapping.volume-type: standard
+        image-type: machine
+      instance_type: t2.micro
+    transport:
+      username: admin
+  - name: debian-6
+    driver:
+      image_search:
+        name: debian-squeeze-*
+        owner-id: 379101102735
+        architecture: x86_64
+        virtualization-type: paravirtual
+        block-device-mapping.volume-type: standard
+        image-type: machine
+      instance_type: t1.micro
```

Comment:
The reason for `t1.micro` is debian-6 AMIs are only available in `paravirtual`, which does not support `t2.micro`. I wanted to keep the defaults at free tier instances, thus `t1.micro`. Sucks.